internal/pkg/tfconf/module.go

```diff
@@ -120,6 +123,26 @@ func CreateModule(path string) (*Module, error) {
 		}
 	}
 
+	//TfOutput ....
```

Comment:
You can make that struct private (`terraformOutput` instead of `TerraformOutput`) and remove the comment